multi/stm32l4-multi/libmulti/libuart.c

```diff
@@ -59,21 +60,42 @@ static const struct {
 };
 
 
+static inline size_t increment_wrap(size_t value, size_t size)
```

Comment:
ALl function names should start with module name.
I'd suggest
`libuart_incMod` or `libuart_incWrap` on `libuart_incrementWrap`

Reply:
I agree that it shouldn't have underscore (incrementWrap)
But I don't like idea to add libuart prefix here as it's static inline helper that should never be exposed in libuart API.
We have [some](https://github.com/phoenix-rtos/phoenix-rtos-devices/blob/a9cbd88f96e5e2380c1f48f23b5363d8a4712440/storage/imxrt-flash/flashsrv.c#L853C21-L853C21) [examples](https://github.com/phoenix-rtos/phoenix-rtos-devices/blob/a9cbd88f96e5e2380c1f48f23b5363d8a4712440/multi/imxrt-multi/cm4.c#L83) in current codebase that we not always do it with prefixes. 

I also think that note in coding convention allowing to omit subsystem  

> Functions used internally in C file should be declared as static. Functions used only inside the selected subsystem could be named with the name of the module instead of the name of subsystem.

Makes me thing that's it's valid to have it as `incrementWrap` without libuart prefix

What do you think @agkaminski ?

Reply:
In this statement it is still required to use some prefix, even for `static`s.

I guess it doesn't really matter that much for a static function. Using one common naming scheme makes it easier to evolve the code, i.e. no need to rename function if we in the end decide to share it and remove `static`. That's the pretty much only objective factor here imho